android/app/src/main/java/com/example/bigwheels/MainActivity.java

```diff
@@ -0,0 +1,32 @@
+package com.example.bigwheels;
+
+import android.view.View;
+
+import com.google.androidgamesdk.GameActivity;
+
+public class MainActivity extends GameActivity {
+    static {
+        System.loadLibrary("bigwheels");
```

Comment:
out of curiosity, would you be able to debug C++ part in VS2022? or just the Java part in Android Studio?

Reply:
I've debugged the c++ part from within Android Studio. It works.
I've never tried VS2022.